.travis.yml

```
@@ -1,15 +1,20 @@
+dist: trusty
 language: go
+go:
+  - '1.11.x'
+  - '1.12.x'
+
 env:
   global:
     - LD_LIBRARY_PATH="/usr/local/lib":${LD_LIBRARY_PATH}
     - GO111MODULE=on
-addons:
-  apt:
-    packages:
-      - libonig-dev
+    - ONIGURUMA_VERSION='6.9.1'
 
-jobs:
-  include:
-    - go: 1.11.x
-      script:
-        - go test -v --cover -race
\ No newline at end of file
+before_install: # install oniguruma manualy as trusty has only ancent 5.x
```

s/ancent/ancient/

---

```suggestions
before_install: # install oniguruma manually as trusty has only ancient 5.x
```

oh, suggestions do not work in reply

---

addressed in d1a3c51